examples/day_in_history.js

```diff
@@ -52,15 +52,20 @@ function ingestArticleProfile(hatch, uri) {
             dayInHistoryYear: dayInHistoryYearFinal
         });
 
+        // TODO: Convert to v2.0 API
         asset.set_document(content);
         asset.set_section("day_in_history");
 
         hatch.save_asset(asset);
+    })
+    .catch(err => {
+        console.error(err.stack);
+        throw err;
     });
 }
 
 function main() {
-    const hatch = new libingester.Hatch();
+    const hatch = new libingester.Hatch('day_in_history', 'en', { argv: process.argv.slice(2) });
```

Comment:
hmm, I didn't realize this was how we were suggesting people to write hatch arguments. this seems very prone to being forgotten/messed up -- IMO argparsing should always only be handled by the "main" function, and then turned into normal function arguments elsewhere. libingester could provide this I guess